### DISCUSSION TITLE
TFT_eSPI and Adafruit touchscreens and class TFT_eSPI has no member named 'getTouch'

### DISCUSSION THREAD

**Discussion by markprince1** [Bodmer/TFT_eSPI#2777] [Q&A - Touch]
Hi 

Great library, but I'm banging my head against a brick wall after an age of searching for an answer. I have an Adafruit 480x320 resistive touchscreen, which (IIRC) has the the STMPE610 driver. I've tested the screen, and I've got the touchscreen working with the Adafruit library, so I don't think I have any setup issues.

I notice there's a chip select pin definition but of course, this screen (pretty much all the Adafruit screens) don't have an SPI touchscreen. 

I don't have the CS pin set to any value, hence getting the "no member named getTouch" issue. Do you have any pointers as to how I can make this work with TFT_eSPI (I'm building an app in SquareLine studio) so I want to keep things as unmodified as possible. I can't see anything in the config that allows me to specify which driver.

Please ... any ideas or config changes to help me bring these two things together?

Thanks

**Comment by Bodmer** [comment]
Do not define TOUCH_CS in the TFT_eSPI setup file. Get the screen working then add the touch library tou the sketch and use the Adafruit touch functions. See their examples.

I am not familiar with the SquareLine studio output so am unable to help on that.
